test/python/test_tf2ngraph_script.py

```diff
@@ -89,12 +96,19 @@ def test_command_line_api(self, inp_format, inp_loc, out_format,
                                     shape_hints)
             if commandline:
                 # In CI this test is expected to be run out of artifacts/test/python
-                command_executor(
-                    'python ../../tools/tf2ngraph.py --input_' + inp_format +
-                    ' ' + inp_loc + ' --output_nodes out_node --output_' +
-                    out_format + ' ' + out_loc + ' --ng_backend ' + ng_device +
-                    ' --config_file ' + config_file_name +
-                    ("", " --precompile ")[precompile])
+                # out_node_str is empty if out_node_name is None.
+                # Automatic output node inference display diagnostic logs
+                # But the tf2ngraph call will still fail
+                out_node_str = ' ' if out_node_name is None else ' --output_nodes ' + out_node_name + ' '
+                try:
+                    command_executor(
+                        'python ../../tools/tf2ngraph.py --input_' +
```

Comment:
This is a bad idea to hardcode relative paths. It makes directories non-portable. Rather use cmake install step to copy the tf2ngraph.py to this directory and run the test.

Reply:
cmake creates the following directory structure (mimicking what we have in the original source tree)

artifacts:
  -test
    -python
      -test_tf2ngraph_script.py
  -tools
    -tf2ngraph.py


the `../../tools/tf2ngraph.py` is needed because pytests run from `artifacts/test/python` and `../../tools/tf2ngraph.py` accesses `artifacts/tools`

The relative path works inside artifacts's copy of tf2ngraph.py (and does not access the original source tf2ngraph.py)